satpy/writers/cf_writer.py

```diff
@@ -71,26 +71,42 @@
 
     >>> my_encoding = {
     ...    'my_dataset_1': {
-    ...        'zlib': True,
+    ...        'compression': 'zlib',
     ...        'complevel': 9,
     ...        'scale_factor': 0.01,
     ...        'add_offset': 100,
     ...        'dtype': np.int16
     ...     },
     ...    'my_dataset_2': {
-    ...        'zlib': False
+    ...        'compression': None,
+    ...        'dtype': np.float64
     ...     }
     ... }
     >>> scn.save_datasets(writer='cf', filename='encoding_test.nc', encoding=my_encoding)
 
 
 2) Via the ``encoding`` attribute of the datasets in a scene. For example
 
-    >>> scn['my_dataset'].encoding = {'zlib': False}
+    >>> scn['my_dataset'].encoding = {'compression': 'zlib'}
     >>> scn.save_datasets(writer='cf', filename='encoding_test.nc')
 
 See the `xarray encoding documentation`_ for all encoding options.
 
+.. note::
+
+    Chunk-based compression can be spezified with the ``compression`` keyword
```

Comment:
```suggestion
    Chunk-based compression can be specified with the ``compression`` keyword
```

Reply:
Good catch, that's a very German mistake :sweat_smile: